### PR TITLE
20 or so fairy tale descriptions

### PR DESCRIPTION
**Pull request by Jerimee**
#### Summary
```SUMMARY: Content "via book/misc.json added more than 20 different descriptions for fairy_tales books"```

#### Purpose of change
```Per the "List of easy jobs" I've added snippet descriptions to fairy tale books in the misc.json file.```

#### Testing
```I ran the game and spawned several "book of fairy tales." Indeed the descriptions were different. I also ran Jsonlint.```

#### Additional context
```
One might expect the snippet ids to be sequential. Mine are unique but not sequential. Instead I used the snippet id to include meta data. The numbers correspond to the Aarne-Thompson-Uther System of folklore taxonomy.  

https://en.wikipedia.org/wiki/Aarne%E2%80%93Thompson%E2%80%93Uther_Index
```

### CODE REVIEW

File: data/json/items/book/misc.json [hunk 1]
```diff
@@ -117,7 +117,84 @@
...
+      {
+        "id": "fairyat_530",
+        "text": "This charming book of Swedish fables is titled, \"The Glass Mountain and the Princess.\""
+      },
+      {
+        "id": "fairyat_555",
+        "text": "This is a collection of fairy tale stories warning against the consequences of extreme greed."
+      },
+      { "id": "fairyat_591", "text": "This book is titled, \"The Thieving Pot: Folktales of the Arab World.\"" },
+      { "id": "fairyat_653", "text": "This is a book of legends collected by Traveller Johnny Cassidy in 1960s." },
```
Comment (I-am-Erk) [inline data/json/items/book/misc.json]:
```suggestion
      { "id": "fairyat_653", "text": "This is a book of legends collected by Traveller Johnny Cassidy in the 1960s." },
```

File: data/json/items/book/misc.json [hunk 2]
```diff
@@ -117,7 +117,84 @@
...
+      { "id": "fairyat_758", "text": "A book by the Brothers Grimm titled, \"Eve's Unequal Children.\"" },
+      { "id": "fairyat_766", "text": "This book of fables expands upon the legend of the Seven Sleepers of Ephesus." },
+      {
+        "id": "fairyat_1060",
+        "text": "In this fairy tale a strong man frightens an ogre by squeezing water out of a stone."
+      },
+      {
+        "id": "fairyat_1084",
+        "text": "This book of rustic folk tales bares the title: \"How to Shout Down the Devil.\""
```
Comment (I-am-Erk) [inline data/json/items/book/misc.json]:
```suggestion
        "text": "This book of rustic folk tales bears the title: \"How to Shout Down the Devil.\""
```

File: data/json/items/book/misc.json [hunk 3]
```diff
@@ -117,7 +117,84 @@
...
+        "id": "fairyat_1287",
+        "text": "The title of this book is \"Village Folk-tales of Ceylon.\"  It includes fables about logical errors and foolish misjudgements of the Kadambawa men."
+      },
+      {
+        "id": "fairyat_1461",
+        "text": "This book of folk tales is titled, \"The Girl with the Ugly Name, and Other Stories.\""
+      },
+      {
+        "id": "fairyat_2025",
+        "text": "Titled, \"The Fleeing Pancake\", this collection of silly folk tales is suitable for small children."
```
Comment (I-am-Erk) [inline data/json/items/book/misc.json]:
```suggestion
        "text": "Titled \"The Fleeing Pancake\", this collection of silly folk tales is suitable for small children."
```

File: data/json/items/book/misc.json (hunk 2 above)
Comment (Jerimee) [reply data/json/items/book/misc.json]: embarrassing! I thought I looked this up, but I must have been confused - thanks for fixing it!

**Comment by Jerimee** [comment]
Thanks @I-am-Erk ! I appreciate you

**Pull request merged**